src/components/Tool/GoogleImport/GoogleImport.tsx

```diff
@@ -0,0 +1,553 @@
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'center',
+  width: '100%',
+}));
+
+const HeaderBox = styled(Box)(({ theme }) => ({
+  backgroundColor: theme.palette.cruGrayLight.main,
+  width: '100%',
+  padding: theme.spacing(1),
+  display: 'flex',
+  justifyContent: 'end',
+  borderTopRightRadius: '7px',
+  borderTopLeftRadius: '7px',
+  [theme.breakpoints.down('sm')]: {
+    paddingTop: theme.spacing(2),
+  },
+}));
+
+const BorderBox = styled(Box)(({ theme }) => ({
+  marginTop: theme.spacing(2),
+  marginBottom: theme.spacing(2),
+  border: `1px solid ${theme.palette.cruGrayLight.main}`,
+  width: '100%',
+  borderRadius: '7px',
+}));
+
+const Section = styled(Box)(({ theme }) => ({
+  marginTop: theme.spacing(2),
+}));
+
+const useStyles = makeStyles()(() => ({
+  bulletList: {
+    margin: '0 0 10px 15px',
+    '> li': {
+      margin: '5px',
+    },
+  },
+  divider: {
+    margin: theme.spacing(2, 0),
+  },
+  headerTitle: {
+    fontSize: '1rem!important',
+  },
+}));
+
+const googleImportSchema = yup.object({
+  tagsForAllList: yup.array().of(yup.string()).default([]),
+  override: yup.string().required(),
+  importAll: yup.string().required(),
+  groupTags: yup.object(),
+  groups: yup.array().of(yup.string()).default([]),
+});
+
+type Attributes = yup.InferType<typeof googleImportSchema>;
+
+interface Props {
+  accountListId: string;
+}
+
+const GoogleImport: React.FC<Props> = ({ accountListId }: Props) => {
+  const { classes } = useStyles();
+  const { appName } = useGetAppSettings();
+  const { t } = useTranslation();
+  const [showModal, setShowModal] = useState(false);
+  const [showConfirmAllModal, setShowConfirmAllModal] = useState(false);
+  const [redirecting, setRedirecting] = useState(false);
+  const [selectedAccount, setSelectedAccount] = useState('');
+
+  const { data: contactTagsList, loading: contactTagsListLoading } =
+    useGetContactTagListQuery({
+      variables: {
+        accountListId,
+      },
+    });
+
+  const { data, loading } = useGoogleContactGroupsQuery();
+  const googleAccounts = data?.googleAccounts;
+
+  const account = useMemo(() => {
+    return googleAccounts?.find((account) => account?.id === selectedAccount);
+  }, [googleAccounts, selectedAccount]);
+
+  const initialValues = useMemo(() => {
+    const initialGroupTags =
+      account?.contactGroups.reduce((acc, group) => {
+        if (group?.id) {acc[group?.id] = [group?.tag];}
+        return acc;
+      }, {}) || {};
+    return {
+      tagsForAllList: [],
+      override: 'false',
+      importAll: 'false',
+      groupTags: initialGroupTags,
+      groups: [] as string[],
+    };
+  }, [account]);
+
+  useEffect(() => {
+    googleAccounts?.length && setSelectedAccount(googleAccounts[0]?.id || '');
+  }, [googleAccounts]);
+
+  const handleCloseModal = () => {
+    setShowModal(false);
+    window.location.href = `${process.env.SITE_URL}/accountLists/${accountListId}/tools`;
```

Comment:
You don't need to explicitly set the domain here.

```suggestion
    window.location.href = `/accountLists/${accountListId}/tools`;
```